lua/neo-tree/sources/common/file-items.lua

```diff
@@ -123,7 +123,9 @@ function create_item(context, path, _type)
     end
     if f.hide_by_pattern then
       for _, p in ipairs(f.hide_by_pattern) do
-        if string.find(name, p) then
+        local separator_pattern = utils.is_windows and '\\' or '/'
```

Comment:
There's just one silly stylua check issue:
```suggestion
        local separator_pattern = utils.is_windows and "\\" or "/"
```